ups/obs_cfht.table

```diff
@@ -2,6 +2,7 @@ setupRequired(daf_persistence)
 setupRequired(obs_base)
 setupRequired(meas_algorithms)
 setupRequired(pipe_tasks)
+setupRequired(astropy)
```

Comment:
I forgot to check for this in the other pyfits->astropy pull requests I reviewed.

Reply:
I've only been adding if the code in bin.src or python/ uses astropy. Not if the examples do.